python/wdl/main.py

```diff
@@ -56,11 +59,17 @@ def cli():
 
     if cli.action == 'run':
         inputs = {}
+        run_service_name = "local"
+
+        if cli.sge:
+            run_service_name = "sge"
+
```

Comment:
Not that I care that much, but `--backend=[local|sge]` would probably be how I'd set this up.  It's not really an issue until there are more backends implemented